### DISCUSSION TITLE
Small bug in UI

### DISCUSSION THREAD

**Discussion by razumv** [subquery/subql#970] [Frontier TestNet - Bug Reporting]
I also noticed a small bug, the display of 20\50\100 indexers per page does not work on the site https://frontier.subquery.network/staking/indexers. It would still add a global filter to search by indexer name. 

![image](https://user-images.githubusercontent.com/42905535/166696022-6bb9a251-eabd-4254-b1af-fa831c218f37.png)

**Comment by mzxyz** [comment]
Thanks for the suggestion, [tikcet: 398]

**Comment by lilian79511** [comment]
Page display issue is fixed. It now supports 10 & 20 per page.
